3 3 parameters selection and inverse model computing

JinsenLou opened the pull request:
Finish the inverse model computing module and the creation of GUI/ connection of model and UI.

JinsenLou committed:
Write the function 'start_inversion'

For the Inversion window, write a start_inversion function and connect to the 'Apply' push button.

JinsenLou committed:
Add the test file for inversion

Add a test data file for function 'start_inversion'.

JinsenLou committed:
Beautify Inversion MainWindow & Set value limits

1. Embedded the resistivity distribution generated by 'invert' into the MainWindow.
2. Adjusted the font size and size of the 'Iterations,' 'Lambda,' and 'dPhi' text boxes.
3. Set the maximum and minimum values for the 'Iterations,' 'Lambda,' and 'dPhi' parameters.

JinsenLou committed:
Add savaFig function

Add the saveFig function for the push button of Save in the inversion window by using the resistivity distribution figure generated by updated startInversion function

wintelestr commented on Inverse Model Computing/soil_gui.py:
It looks great, good logic and structure of codes.

zhangqianalone replied:
Good structure code with clear comments and the function is well built.

wintelestr reviewed (commented):
Great Job!

JinsenLou committed:
Add the user manual for Inversion window

Add the user manual/ instruction for the inversion window for explaining the 3 spin boxes and 2 push buttons

dtdoris commented:
>  Code section for inversion is well commented and explained.

JinsenLou committed:
Improve the ReadMe.md for Inversion

Add pictures for better explaining steps.
Modify the steps to reduce ambiguity.

JinsenLou committed:
Add the pictures of inversion window

Add 3 pictures of the inversion window for creating the link to ReadMe

JinsenLou committed:
Improve the ReadMe for inversion module

Make some small changes for the ReadMe.md of this module

zhangqianalone reviewed (approved):
good structure codes with clear comments

9264Y commented:
The code has a very clean and organised structure.
